examples/README.md

```diff
@@ -61,6 +61,9 @@ Uses the [`iro.js`](https://iro.js.org) JavaScript library to showcase a user in
 ### [`contexttypesbot.py`](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/contexttypesbot.py)
 This example showcases how `telegram.ext.ContextTypes` can be used to customize the `context` argument of handler and job callbacks.
 
+### [`customwebhookbot.py`](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/customwebhookbot.py)
+This example showcases how a custom webhook setup can be used in combination with `telegram.ext.Application`.
```

Comment:
```suggestion
This example showcases how a custom webhook setup can be used in combination with `telegram.ext.Application`. Also contains an health endpoint implementation.
```

Reply:
mh, "custom webhook setup" already includes arbitrary endpoints. I see no benefit of this note …

Reply:
alright, thought about mentioning it since that issue was open for so long and had quite a few thumbs ups, but Im fine with not mentioning it

Reply:
We'll close #1438 by pointing to the example, I guess that should suffice :)